tests/Controllers/Multiplayer/RoomsControllerTest.php

```diff
@@ -429,6 +429,100 @@ public function testJoinWithPassword()
         $this->assertSame($initialUserChannelCount + 1, UserChannel::count());
     }
 
+    public function testDestroy()
+    {
+        $token = Token::factory()->create(['scopes' => ['*']]);
+        $start = now();
+        $end = $start->addMinutes(60);
```

Comment:
the object created by `now()` mutates itself so the start and end are both the same. Either `clone()` it first or use `Carbon\CarbonImmutable`